assets/js/modules/analytics/components/setup/SetupMain.js

```diff
@@ -96,8 +98,11 @@ export default function SetupMain( { finishSetup } ) {
 	useExistingTagEffect();
 
 	useEffect( () => {
-		trackEvent( 'analytics_setup', 'configure_analytics_screen' );
-	}, [] );
+		trackEvent(
+			`${ viewContext }_analytics`,
+			'configure_analytics_screen'
+		);
+	}, [ viewContext ] );
```

Comment:
Same here, @felixarntz do we need to change the event name here?

Reply:
@eugene-manuilov @ivankruchkoff Same here, this should be removed, as there is now the more general `view_module_setup` event.

Reply:
Removed in https://github.com/google/site-kit-wp/pull/4321/commits/ebbe2b8ab57ee87215f28f0395da1268fdf667f2